python_scripts/02_numerical_pipeline_ex_00.py

```diff
@@ -57,7 +46,7 @@
 # Write your code here.
 
 # %% [markdown]
-# Fit this model on the data and target loaded above
+# Fit this model on the train data and target that we used before
```

Comment:
Not sure whether `train data` was a typo here or used intentionally. It sounds a bit strange to me.

```suggestion
# Fit this model on the training data and target that we used before
```

Reply:
After reading the description of #31, I think it should indeed be `training data`.